client/src/components/Modal/Modal.stories.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import Modal from './Modal';
+import "../../App.css";
+
+function handleConfirm(evt) {
+  evt.preventDefault()
+  alert('Confirmed!')
+}
+function handleJoin(evt) {
+  evt.preventDefault()
+  alert('Joined!')
+}
+function handleCancel (evt){
+  evt.preventDefault()
+  alert('Canceled!')
+}
+function handleClose (evt){
+  evt.preventDefault()
+  alert('Modal Closed!')
+}
+
+storiesOf('Modal', module)
+  .add("Follow Modal", () => 
+    <Modal message={"Are you sure you want to follow"} 
+          confirmText={"Confirm"} 
+          cancelText={"Cancel"}
+          onConfirm={handleConfirm} 
+          onCancel={handleCancel}
```

Comment:
I think `onCancel` and `onClose` are the same - we should invoke the same callback when they click Cancel as when they click the X